src/App.js

```diff
@@ -338,7 +339,13 @@ function Root() {
   return initializing ? null : (
     <ReduxProvider store={store}>
       <RecoilRoot>
-        <PersistQueryClientProvider client={queryClient} persistOptions={persistOptions}>
+        <PersistQueryClientProvider
+          client={queryClient}
+          persistOptions={persistOptions}
+          onSuccess={() => {
+            prefetchDefaultFavorites();
+          }}
```

Comment:
run this after hydrating the cache, so we can skip this fetch if there is data stored already